docs/CONFIG-PROPERTIES.md

```diff
@@ -12,7 +12,7 @@
 | timer.use.config.checkpoint | integer in seconds | 600 | use checkpointed config if no cloud connectivity |
 | timer.gc.download | integer in seconds |  600 | garbage collect unused downloaded objects |
 | timer.gc.vdisk | integer in seconds | 1 hour | garbage collect unused instance virtual disk |
-| timer.gc.rkt.graceperiod | integer in seconds | 300 | grace period to be used in rkt gc command |
+| timer.gc.rkt.graceperiod | integer in seconds | 86400 | grace period to be used in rkt gc command |
```

Comment:
As discussed, make the default value 1 hour

Reply:
Resolved.